<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationSetupWidget</class>
 <widget class="QWidget" name="LocationSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>852</width>
        <height>312</height>
       </rect>
      </property>
      <attribute name="label">
       <string>System Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPushButton" name="pbRestartRds">
         <property name="text">
          <string>Restart RDS</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Working Platforms</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutPlatforms"/>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="SystemSettingsWidget" name="systemSettingsWgt" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbGetSystem">
         <property name="text">
          <string>Get system Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>835</width>
        <height>429</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Mode Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Location</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sbDuration">
            <property name="suffix">
             <string comment="ms"> ms</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Central frequency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="sbCentralFrequency">
            <property name="suffix">
             <string> mhz</string>
            </property>
            <property name="minimum">
             <number>20</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Convolution</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="cbConvolution">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Doppler</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="cbDoppler">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Reveive Convolution</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="cbGetConvolution">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Average frequency</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSpinBox" name="sbAverageFreq">
            <property name="suffix">
             <string> khz</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>7000</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Selection</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QDoubleSpinBox" name="sbSelStart">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>End</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QDoubleSpinBox" name="sbSelEnd">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="16" column="0" colspan="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="17" column="0" colspan="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Analysis</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="18" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item row="18" column="1">
           <widget class="QSpinBox" name="sbAnalysisDuration">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
           </widget>
          </item>
          <item row="20" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="21" column="0" colspan="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Record Signal</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="23" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Diration</string>
            </property>
           </widget>
          </item>
          <item row="23" column="1">
           <widget class="QSpinBox" name="sbRecordDuration">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="cbHumps">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Humps</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>852</width>
        <height>312</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Rds error log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SystemSettingsWidget</class>
   <extends>QWidget</extends>
   <header>SystemSettings/SystemSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
